back-office/multistoregeneralspecs.md

```diff
@@ -0,0 +1,199 @@
+The shop association block displays all groups and all shops.<br/>
+Checkboxes allow to select / unselect groups and shops.<br/>
+It's possible to expand / collapse the wall tree and expand / collapse each group<br/>
+If you select / unselect a group, all shops of this group are selected / unselected
+
+The shop association block allows to choose for which store(s) you want to associate or disassocate the current element.
+
+When you add something, the shop(s) selected in the context are automatically checked in the shop association block.<br/>
+But it's possible to select / uncelect shops, no matter the context selected.
+
+When you edit something, you can unselect shop(s) previously selected and therefore disassocate it from these shops.<br/>
+And you can also, select shop(s) not previously selected and therefore associate it with these shops.<br/>
+If the merchant unselect all the checkboxes (all the shops), then an alert message in red is displayed above or below the shop association block to inform of the obligation to have at least one shop checked: "You should select at least one shop to be able to save" + the save button is disabled. **only from 1.7.8**
+
+**Add / edit pages with shop association block**
+
+- Catalog > Categories > Add / edit category
+- Catalog > Attributes > Attributes group > Add / edit
+- Catalog > Attributes > Attributes value > Add / edit
+- Catalog > Features > Feature > Add / edit feature
+- Catalog > Brands > Add / edit brand
+- Catalog > Suppliers > Add / edit supplier
+- Design > Pages > Add / edit page
+- Design > Pages > Add / edit category page
+- Shipping > Carriers > Add / edit carrier
+- International > Localization > Languages > Add / edit
+- International > Localization > Currencies > Add / edit
+- International > Locations > Zones > Add / edit
+- International > Locations > Countries > Add / edit
+- International > Taxes > Taxes rules > Add / edit
+- Shop parameters > Customer settings > Groups > Add / edit
+- Shop parameters > Contact > Contacts > Add / edit
+- Shop parameters > Contact > Stores > Add / edit
+- Shop parameters > Traffic & SEO > Referrers > Add / edit
+- Advanced parameters > Webservice > Add / edit
+
+
+### Actions in the lisitngs (grids)
+
+**Delete action**
+
+If an element (category, attribute, feature ...) is associated with several shops, then if I delete it from the listing, it will be disassociated for the selected context(s) and deleted if the all shops context is selected. 
+
+**Enable/disable**
+
+If an element (category, attribute, feature ...) is associated with several shops, then if I enable or disable it from the listing, the action will be done for the store(s) with which the item is associated and not only for the selected context. 
+When the user hovers the enable/disable icon, a tooltip is displayed "This action will be applied to the store(s) with which the item is associated" (**from 1.7.8**)
```

Comment:
@LouiseBonnard let me know if the message is clear enough ?

Reply:
I would keep it short and stay consistent with the wording mentioned [here](https://docs.google.com/document/d/1tVzwbxq-gNyfpBbIVSL1yUuQxwO6HLuMh8u26VFPESk/edit) with a tooltip like _It will apply to all stores associated with this item._, what do you think?

Reply:
@LouiseBonnard Ok for this one !

Reply:
👌